Dockerfile

```diff
@@ -23,10 +23,12 @@ RUN mkdir /projects ${HOME} && \
 ENV GLIBC_VERSION=2.31-r0 \
     ODO_VERSION=v1.2.1 \
     OC_VERSION=4.3.3 \
-    KUBECTL_VERSION=v1.18.3
+    KUBECTL_VERSION=v1.18.3 \
+    KUBECTX_VERSION=0.8.0
 
-# plugin executes the commands relying on Bash
-RUN apk add --update --no-cache bash && \
+RUN apk add --update --no-cache curl git jq py-pip bash bash-completion ncurses pkgconfig && \
+    ln -sf /bin/bash /bin/sh && \
+    echo "source /etc/profile.d/bash_completion.sh" >> ~/.bashrc && \
     # install glibc compatibility layer package for Alpine Linux
     # see https://github.com/openshift/origin/issues/18942 for the details
     wget -O glibc-${GLIBC_VERSION}.apk https://github.com/andyshinn/alpine-pkg-glibc/releases/download/${GLIBC_VERSION}/glibc-${GLIBC_VERSION}.apk && \
```

Comment:
for which tool do we need glibc ?

Reply:
It's a good question for author of that line ) I have no idea